test-integration/helpers/select-readings-dialog.coffee

```diff
@@ -0,0 +1,51 @@
+{TestHelper} = require './test-element'
+
+OPENED_PANEL_SELECTOR = '.dialog:not(.hide)'
+
+COMMON_SELECT_READING_ELEMENTS =
+  showProblemsBtn:
+    css: '.homework-plan-exercise-select-topics button.-show-problems'
+
+  sectionItem: (section) ->
+    css: "#{OPENED_PANEL_SELECTOR} [data-chapter-section='#{section}']"
+  chapterHeadingSelectAll: (section) ->
+    css: "#{OPENED_PANEL_SELECTOR} [data-chapter-section='#{section.split('.')[0]}'] .chapter-checkbox input"
+  chapterHeading: (section) ->
+    css: "#{OPENED_PANEL_SELECTOR} [data-chapter-section='#{section.split('.')[0]}'] > a"
+  loadingLocator:
+    css: '.loadable.is-loading'
+
+  nextStepBtn:
+    css: '.panel-footer .btn.btn-primary'
+
+class SelectReadingsList extends TestHelper
+  constructor: (test, testElementLocator) ->
+    testElementLocator ?= css: ".select-reading-dialog#{OPENED_PANEL_SELECTOR}"
+    super test, testElementLocator, COMMON_SELECT_READING_ELEMENTS, defaultWaitTime: 5000
```

Comment:
I increased it just because it was always timing out on loading that particular element, and this was the only place I saw that could do it, otherwise, we'll have to put in that timeout every time we load this element.